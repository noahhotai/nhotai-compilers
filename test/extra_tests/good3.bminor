
/* returning right type */

testfunc: function void ( x: integer, y:integer, z:integer ) = {
	return testfunc(x, y, z);
}


