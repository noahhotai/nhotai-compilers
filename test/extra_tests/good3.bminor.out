Program parsed successfully.
testfunc resolves to global testfunc
x resolves to param 1
y resolves to param 2
z resolves to param 3
Program resolved successfully.
paramater error: wrong of arguments given to the function testfunc.
Program type checked successfully.
